pkg/app/master/commands/build/cli.go

```diff
@@ -167,7 +167,8 @@ var CLI = &cli.Command{
 		commands.Cflag(commands.FlagSensorIPCMode),
 	},
 	Action: func(ctx *cli.Context) error {
-		xc := app.NewExecutionContext(Name)
+		gparams, ok := commands.CLIContextGet(ctx.Context, commands.GlobalParams).(*commands.GenericParams)
```

Comment:
Should use `ctx.String(commands.FlagConsoleOutput)` like in other commands to be consistent. Also the moved code to grab generic params from the context here doesn't have the check for `ok`. There's a bit of a chicken and egg problem here. If getting the generic params from the context fails we need to call xc.Out.Error and xc.Out.State.

Reply:
Done, thanks so much for this!